presto-main/src/main/java/com/facebook/presto/sql/planner/QueryPlanner.java

```diff
@@ -290,6 +303,34 @@ private PlanBuilder project(PlanBuilder subPlan, Iterable<Expression> expression
                 analysis.getParameters());
     }
 
+    private Expression rewriteGroupingOperation(PlanBuilder subPlan, Node node, Expression expression)
```

Comment:
What's the point of this? It seems like a copy of the same method in GroupingOperationRewriter.